src/agent/onefuzz-supervisor/src/heartbeat.rs

```diff
@@ -54,3 +54,32 @@ pub async fn init_agent_heartbeat(queue_url: Url) -> Result<AgentHeartbeatClient
     );
     Ok(hb)
 }
+
+pub trait HeartbeatSender {
+    fn send(&self, data: HeartbeatData) -> Result<()>;
+
+    fn alive(&self) {
+        self.send(HeartbeatData::MachineAlive).unwrap()
```

Comment:
I suggest _not_ unwrapping here, or anywhere else in the heartbeat code. If heartbeat sending fails, we don't want it to kill the agent.